Lib/functools.py

```diff
@@ -238,12 +238,14 @@ def reduce(function, sequence, initial=_initial_missing):
     """
     reduce(function, iterable[, initial], /) -> value
 
-    Apply a function of two arguments cumulatively to the items of a sequence
-    or iterable, from left to right, so as to reduce the iterable to a single
-    value.  For example, reduce(lambda x, y: x+y, [1, 2, 3, 4, 5]) calculates
-    ((((1+2)+3)+4)+5).  If initial is present, it is placed before the items
-    of the iterable in the calculation, and serves as a default when the
-    iterable is empty.
+    Apply a function of two arguments cumulatively to an iterable, from left to right.
```

Comment:
Removing 'sequence' is IMO ok, since a sequence is an iterable, but I wonder if we should be more aligned to the original wording here. How about this:

```suggestion
    Apply a function of two arguments cumulatively to the items of an iterable, from left to right.
```

Reply:
I'm ok with this, but line will be too long.  Is there a chance to misunderstood "shortened" version?

Reply:
> [...] but line will be too long.

For docstring summary lines, I think this is ok.

> Is there a chance to misunderstood "shortened" version?

Good question. Perhaps the @python/editorial-board can chime in.

Reply:
I think we should go for the long version. Let's apply it and land this.

Reply:
I think the long version is fine. You could slightly reduce the length by:

`Apply a function of two arguments cumulatively to an iterable's items, from left to right.`

Though I don't think it reads any better.